repository-search/search-by-access-type/demo.js

```diff
@@ -0,0 +1,47 @@
+visualize({
+    auth: {
+      name: "joeuser",
+      password: "joeuser",
+         organization: "organization_1"
+    }
+  }, function(v) {
+    var accessTypeElement = document.getElementById('accessType');
```

Comment:
I would not mix `let` and `var` in the same code unless it is really necessary. 
Please replace all the occurrences of `var` with either `let` or `const` based on the context.